mslice/scripting/__init__.py

```diff
@@ -0,0 +1,13 @@
+from mantid.simpleapi import GeneratePythonScript
+from mslice.models.workspacemanager.workspace_provider import get_workspace_handle
+from mslice.util.qt import QtWidgets
+
+
+def generate_script(plot_window, ws_name, filename):
+    if filename is None:
+        path = QtWidgets.QFileDialog.getSaveFileName(plot_window, 'Save File', '', 'Python (.py)')
+        if path is u'':
```

Comment:
It is slightly more pythonic to have an empty string check like:

```
if not path:
```

https://stackoverflow.com/questions/9573244/most-elegant-way-to-check-if-the-string-is-empty-in-python